docs/instant.md

```diff
@@ -562,11 +562,20 @@ The combination of `roundingIncrement` and `smallestUnit` must make an increment
 
 The `roundingMode` option controls how the rounding is performed.
 
-- `ceil`: Always round up, towards the end of time.
-- `floor`, `trunc`: Always round down, towards the beginning of time.
-  (These two modes behave the same, but are both included for consistency with `Temporal.Duration.round()`, where they are not the same.)
+- `ceil`: Always round up, towards the end of time (positive infinity).
+- `expand`:  Always round away from from zero like `ceil` for positive value and like `floor` for negative value.
+- `floor`: Always round down, towards the beginning of time (negative infinity).
+- `trunc`: Always round towards zero.
+- `halfCeil`: Round to the nearest of the values allowed by `roundingIncrement` and `smallestUnit`.
+  When there is a tie, round towards positive infinity like `ceil`.
 - `halfExpand`: Round to the nearest of the values allowed by `roundingIncrement` and `smallestUnit`.
-  When there is a tie, round up, like `ceil`.
+  When there is a tie, round away from zero, like `expand`.
+- `halfFloor`: Round to the nearest of the values allowed by `roundingIncrement` and `smallestUnit`.
+  When there is a tie, round towards negative infinity like `floor`.
+- `halfTrunc`: Round to the nearest of the values allowed by `roundingIncrement` and `smallestUnit`.
+  When there is a tie, round towards zero like `trunc`.
+- `halfEven`: Round to the nearest of the values allowed by `roundingIncrement` and `smallestUnit`.
+  When there is a tie, round towards even cardinality.
```

Comment:
"cardinality" is a pretty technical term that many developers may not be familiar with. Would it be accurate to say "towards the closest even number" instead? If yes, then let's use that wording here and in other places where "cardinality" is used.

Reply:
I chose the word "cardinality" in the NFv3 spec because the number is not necessarily even if a roundingIncrement is used. If Temporal doesn't support roundingIncrement, then you can use other language.

Reply:
Oh, interesting. How does halfEven and roundingIncrement interact?  What does "even" mean in that context?  Even number of increments?

Reply:
If roundingIncrement is 2, for example, how do you round 3? 2 or 4 are both even. So you choose the one that has even cardinality, which is 4, or 2 increments up from zero.

Reply:
What about something like "round towards the even number of `roundingIncrement`s" ?

Reply:
Yeah, this is a super difficult one to explain in a documentation-friendly way. Building on Justin's suggestion, how about "When there is a tie, round towards the result which for which the difference is an even multiple of `roundingIncrement`?" Maybe an example would help, additionally.